docs/dev/development/getting-started.rst

```diff
@@ -427,7 +427,7 @@ compilation errors due to your system not including libraries
 or binaries required by some of Warehouse's dependencies.
 
 An example of such dependency is
-`psycopg2 <https://www.psycopg.org/docs/install.html#prerequisites>`_
+`psycopg <http://initd.org/psycopg/docs/install.html#prerequisites>`_
```

Comment:
If we continue along the "Local installation" for psycopg, when use this URL (the initd one is dead no matter what)

```suggestion
`psycopg <https://www.psycopg.org/psycopg3/docs/basic/install.html#local-installation>`_
```